ts/modules/sunburst.src.ts

```diff
@@ -840,14 +840,16 @@ var sunburstOptions: Highcharts.SunburstSeriesOptions = {
 
         /**
          * Decides how the data label will be rotated relative to the perimeter
-         * of the sunburst. Valid values are `auto`, `parallel` and
-         * `perpendicular`. When `auto`, the best fit will be computed for the
-         * point.
+         * of the sunburst. Valid values are `auto`, `circular`, `parallel` and
+         * `perpendicular`. When `auto` or `circular`, the best fit will be
+         * computed for the point, where `circular` use the textPath feature.
```

Comment:
Here we should also explain the pros and cons of the automatic options. In short, the `circular` options have a better looking layout, but they don't support multiple lines and `textOutline`.